app.xmcl.voxelum.yaml

```diff
@@ -0,0 +1,75 @@
+app-id: app.xmcl.voxelum
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: xmcl-wrapper.sh
+build-options:
+  no-debuginfo: true
+finish-args:
+  # Basic System Access
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=home
+  # Media & Communication
+  - --filesystem=xdg-run/pipewire-0:ro
+  - --filesystem=xdg-run/discord-ipc-0:ro
+  # System Integration
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=com.feralinteractive.GameMode
+  # Electron Settings
+  - --env=ELECTRON_ENABLE_STACK_DUMPING=false
+  - --env=ELECTRON_ENABLE_LOGGING=false
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+
+modules:
+  - name: xrandr
+    sources:
+      - type: archive
+        url: https://xorg.freedesktop.org/archive/individual/app/xrandr-1.5.3.tar.xz
+        sha256: f8dd7566adb74147fab9964680b6bbadee87cf406a7fcff51718a5e6949b841c
+    cleanup:
+      - /share/man
+      - /bin/xkeystone
+
+  - name: zypak
```

Comment:
use the electron baseapp.

Reply:
zypak is in the electron baseapp that's why i told you to use it this should be removed